client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.js

```diff
@@ -43,6 +43,12 @@ const DataFilesToolbar = ({ scheme, api }) => {
       state.workbench && state.workbench.config.makeLink && api === 'tapis'
   );
 
+  const showMakePublic = useSelector(state =>
+    state.systems.systemList.find(
+      sys => sys.publishable && sys.scheme === 'public'
+    )
+  );
+
```

Comment:
Need to also check [state.workbench.config](https://github.com/TACC/Frontera-Portal/blob/d8dc5dc6108dfb979393c899258c0ac7ed7d7e8e/client/src/redux/reducers/workbench.reducers.js#L3) to see if makePublic is enabled for the portal.

Reply:
`publishable` is a setting on the system object, so it's controlled through config.

Reply:
I think the idea behind state.workbench.config is to have a single place to enable/disable features that other portals do or don't want.  So for example we have settings for View Path functionality.

Reply:
Settings secret has this, which propagates up to state.workbench.config

```
_WORKBENCH_SETTINGS = {
    "debug": _DEBUG,
    "viewPath": True,
    "makeLink": True
}
```

Reply:
OK, I moved the config into Workbench Settings.